index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="headwrap">
+    <header class="header">
+     <nav class="nav">
+         <i id="menu" class="material-icons">menu</i>
+     </nav>
+    </header>
+    <section class="headline">
+        <h1 class="name">Desmond
+            Wanjala</h1>
+        <p class="p-text">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        <button type="button" class="collabo">Start collaboration</button>
+        <img  class="imaghead" src="/images/headline.svg" alt="">
```

Comment:
- [ ] KIndly place this image properly as the previous reviewers had noted. 😄 

## Your Design
![crr13](https://user-images.githubusercontent.com/103386183/198553271-5be1afbc-10b7-41c1-9a04-07307b573351.png)


## Figma Design 

![crr12](https://user-images.githubusercontent.com/103386183/198553378-d83e7705-909c-4d19-a9cb-c1ab7f65f56d.png)


- [ ] [optional] I am sorry that no reviewer mentioned this, the image should have been placed with CSS background property. 👍